tests/core/eth-module/test_transactions.py

```diff
@@ -41,6 +45,46 @@ def test_send_transaction_with_valid_chain_id(web3, make_chain_id, expect_succes
         assert 'chain ID' in str(exc_info.value)
 
 
+@pytest.mark.parametrize(
+    'to, _from, expect_success',
+    (
+        (
+            'thedao.eth',
+            'notanaddress.eth',
```

Comment:
maybe slightly less confusing if you used something like `not_a_registered_ens_name.eth`.